.github/workflows/lint.yml

```
@@ -30,5 +30,8 @@ jobs:
       - name: Run ESLint
         run: yarn lint:es
 
+      - name: Run tsc (type check)
```

[Minor] Don't mind either, but maybe:

```suggestion
      - name: Run TypeScript compiler (type check)
```